x-pack/plugins/infra/public/components/asset_details/tabs/osquery/osquery.tsx

```diff
@@ -22,13 +24,16 @@ export const Osquery = () => {
 
   // avoids component rerender when resizing the popover
   const content = useMemo(() => {
+    if (!featureFlags.osqueryEnabled) {
+      return <></>;
```

Comment:
It can be just `null`, right? Would save one node in virtual DOM 😅

Reply:
true! will change it.